static/styles/left_sidebar.css

```diff
@@ -155,8 +155,6 @@ li.show-more-topics {
 }
 
 .narrows_panel {
-    margin-bottom: 4px;
-
```

Comment:
The commit message for this commit should be line-wrapped, and use proper English (start sentences with a capital letter, etc.).

I also don't understand why removing this CSS as part of this change is correct.

Reply:
@timabbott , I removed this, once i introduced a margin-bottom for "subscription-link" then this `margin-bottom` CSS had no specific impact on all over the CSS of the page!
I cross-checked it several times, but didn't find any use of this !

Attaching a SS where this margin was coming and putting on an impact earlier when there was no `margin-bottom` for subscription link.

Reply:
![Screenshot from 2022-02-10 03-46-45](https://user-images.githubusercontent.com/76561593/153299855-a30b698d-f336-40b9-88f8-7f12bebb7419.png)

Reply:
and now as you can see, I have given enough `margin-bottom` to `#add-stream-link` so this part of CSS was useless.